documentation/Windows Event Log (EVT) format.asciidoc

```diff
@@ -210,12 +218,11 @@ This value is used to indicate the end of the event record
 |===
 
 [NOTE]
-The "strings offset" points to the offset of "copy of size" (or "data offset"
-if "data size" is not 0) the strings are empty. [yellow-background]*Can the
-"strings offset" contain 0 ?*
+If the user SID size is 0 then the string offset may equal the user SID offset. +
```

Comment:
Trailing `+` is not needed

Reply:
I thought it looked better with the three "if"s on separate lines.